network/gossip/libp2p/test/sparsetopology_test.go

```diff
@@ -1,3 +1,4 @@
+//nolint:unused
```

Comment:
What is the reason this needs to be ignored now?

Reply:
Due to the linter upgrade to 1.29. The new lint is quite smart, and considers code after a `t.Skip()` to be un-used, so this file, where many tests are skipped, now has two "unused" functions and no longer pass lint.

Reply:
Ah, nice, cool!